cmd/auth/main.go

```diff
@@ -52,6 +53,7 @@ const (
 	defKetoHost      = "mainflux-keto"
 	defKetoWritePort = "4467"
 	defKetoReadPort  = "4466"
+	defLoginDuration = "600m"
```

Comment:
Please use `10h`, it's more user-friendly.

Reply:
Done